README.md

```diff
@@ -96,24 +104,51 @@ The SEDA EVM Contracts enable interaction with the SEDA network through two main
    }
    ```
 
+3. **IFeeManager**
+   ```solidity
+   interface IFeeManager {
+       // Adds fees to recipient's pending balance
+       function addPendingFees(address recipient) external payable;
+       
+       // Adds fees for multiple recipients
+       function addPendingFeesMultiple(
+           address[] calldata recipients, 
+           uint256[] calldata amounts
+       ) external payable;
+       
+       // Withdraws accumulated fees
+       function withdrawFees() external;
+       
+       // Gets pending fee balance
+       function getPendingFees(address account) external view returns (uint256);
+   }
+   ```
+
 ### Data Flow
 
 1. **Request Flow**
-   - Users submit requests through `SedaCore.postRequest()`
+   - Users submit requests through `SedaCore.postRequest()` with associated fees
    - Requests are stored and tracked in pending state
    - Each request includes execution and tally parameters
-   - **Incentives**: Users attach a `requestFee` which is used to forward requests to the SEDA network.
+   - **Incentives**: Users attach fees which are held by the FeeManager until distribution:
```

Comment:
This is not true right? The fee is held by the core contract and only during distribution is it sent to the FeeManager.